.yarn/patches/@credo-ts-anoncreds-npm-0.5.13-446ac3168e.patch

```diff
@@ -0,0 +1,82 @@
+diff --git a/build/anoncreds-rs/AnonCredsRsHolderService.js b/build/anoncreds-rs/AnonCredsRsHolderService.js
+index c1eeeca1c9c254f47880909391b22f66598f42fd..b1e0b7913615b2e0b03b8c206ba1f1f02deb54cf 100644
+--- a/build/anoncreds-rs/AnonCredsRsHolderService.js
++++ b/build/anoncreds-rs/AnonCredsRsHolderService.js
+@@ -82,7 +82,8 @@ let AnonCredsRsHolderService = class AnonCredsRsHolderService {
+                         ].join('\n'));
+                     }
+                 }
+-                const { linkSecretId, revocationRegistryId, credentialRevocationId } = (0, w3cAnonCredsUtils_1.getAnoncredsCredentialInfoFromRecord)(credentialRecord, (0, proofRequest_1.proofRequestUsesUnqualifiedIdentifiers)(proofRequest));
++                const proofUsesUnqualifiedIdentifiers = (0, proofRequest_1.proofRequestUsesUnqualifiedIdentifiers)(proofRequest);
++                const { linkSecretId, schemaId, credentialDefinitionId, revocationRegistryId, credentialRevocationId } = (0, w3cAnonCredsUtils_1.getAnoncredsCredentialInfoFromRecord)(credentialRecord, proofUsesUnqualifiedIdentifiers);
+                 // TODO: Check if credential has a revocation registry id (check response from anoncreds-rs API, as it is
+                 // sending back a mandatory string in Credential.revocationRegistryId)
+                 const timestamp = attribute.timestamp;
+@@ -112,6 +113,13 @@ let AnonCredsRsHolderService = class AnonCredsRsHolderService {
+                             credential: credentialRecord.credential,
+                         })
+                         : credentialRecord.credential;
++
++                    if (proofUsesUnqualifiedIdentifiers) {
++                        credential.schema_id = schemaId;
++                        credential.cred_def_id = credentialDefinitionId;
++                        credential.rev_reg_id = credentialRevocationId != null ? credentialRevocationId : undefined;
```

Comment:
```suggestion
+                        credential.rev_reg_id = revocationRegistryId != null ? revocationRegistryId : undefined;
```

Reply:
Error I see in Traction is
```
marshmallow.exceptions.ValidationError: {'identifiers': {0: {'rev_reg_id': ['Value 1565 is not an indy revocation registry identifier']}}}
```
So "1565" probably looks like a credential revocation ID instead of the rev reg ID

Reply:
That was my bad. I think I just saw the RevocationId part and overlooked it was assigning the wrong thing here. I should have tested better this scenario with revocation as well.